observations/obs_converters/AIRS/work/input.nml

```diff
@@ -12,10 +12,12 @@
                              '../../../../observations/forward_operators/obs_def_AIRS_mod.f90'
   /
 
-! version 5 file?:
+! example version 5 file:
 !   l2_files = '../data/AIRS.2007.11.01.001.L2.RetStd.v5.2.2.0.G08078150655.hdf'
-! version 6 file?:
+! example version 6 file:
 !   l2_files = '../data/AIRS.2017.01.01.110.L2.RetStd_IR.v6.0.31.1.G19058124823.hdf'
+! example version 7 file:
+!   l2_files = '../AIRS.2020.06.15.224.L2.RetStd_IR.v7.0.4.0.G20330033505.hdf'
```

Comment:
I think this all looks good. The docs are good and the links all seem to be working.

My one question has to do with these example version files, specifically the paths to them. How come for versions 5 and 6, the path to the l2 file is '../data/' while the path to version 7 file is just '../' ? And then there is no path to the l2 file included in the actual namelist value set below?

In the &convert_amsu_L1_nml , the namelisst value for l1_files uses the path '../data/'

Reply:
@braczka @kdraeder do either of you know about this or have any thoughts?

Reply:
Technically, the path to the raw data files needs to be customized for each application, so I don't have strong feelings whether the template namelist files should include a dummy path to a file or be left blank within the actual l2_file name. 

 With that being said, it is fairly standard for the raw data files to be included in the ../data/ path, so for consistency within /AIRS/work/input.nml, I agree that the example version 7 (like 5 and 6) should also be within the ../data/ directory.  As a side note, it's also a bit non-standard to include documentation/comments within the namelist files themselves -- probably better to include all documentation within the rst files.

Reply:
Sounds good, I agree with all of this and don't think we should have comments like this in an namelist file. I went ahead and made a commit for this.